### ISSUE TITLE
TS enhancement for BIG-IP as pull consumer for 3rd party integrations

### ISSUE THREAD

**Issue by payalsin** [F5Networks/f5-telemetry-streaming#93]
We are looking at collecting basic Stats from the BIG-IP related to the VIP and Pool as we see on the BIG-IP.

![image](https://user-images.githubusercontent.com/13107434/98984529-2dce5e80-24d7-11eb-8e64-6949baa9b93c.png)

Looking to collect info suck as : Bits In/Out, Packets In/Out, Connection, Requests.

From the TS output see the following I see only bitsIn and bitsOut (no Packets, Connections, or Request information)
```json
[
   {
      "virtualServers":{
         "/Dynamic/DemoApp1/serviceMain":{
            "clientside.bitsIn":8867392,
            "clientside.bitsOut":18203840,
            "clientside.curConns":0,
            "destination":"10.168.56.252:80",
            "mr.msgIn":0,
            "mr.msgOut":0,
            "mr.reqIn":0,
            "mr.reqOut":0,
            "mr.respIn":0,
            "mr.respOut":0,
            "availabilityState":"available",
            "enabledState":"enabled",
            "status.statusReason":"The virtual server is available",
            "name":"/Dynamic/DemoApp1/serviceMain",
            "ipProtocol":"tcp",
            "mask":"255.255.255.255",
            "pool":"/Dynamic/DemoApp1/web_pool",
            "profiles":{
               "/Common/f5-tcp-progressive":{
                  "name":"/Common/f5-tcp-progressive"
               },
               "/Common/http":{
                  "name":"/Common/http"
               }
            }
         }
      },
      "pools":{
         "/Dynamic/DemoApp1/web_pool":{
            "activeMemberCnt":18,
            "curPriogrp":0,
            "highestPriogrp":0,
            "lowestPriogrp":0,
            "mr.msgIn":0,
            "mr.msgOut":0,
            "mr.reqIn":0,
            "mr.reqOut":0,
            "mr.respIn":0,
            "mr.respOut":0,
            "serverside.bitsIn":7705920,
            "serverside.bitsOut":15016032,
            "serverside.curConns":0,
            "availabilityState":"available",
            "enabledState":"enabled",
            "status.statusReason":"The pool is available",
            "name":"/Dynamic/DemoApp1/web_pool",
            "members":{
               "/Dynamic/192.168.56.150:80":{
                  "addr":"192.168.56.150",
                  "monitorStatus":"up",
                  "mr.msgIn":0,
                  "mr.msgOut":0,
                  "mr.reqIn":0,
                  "mr.reqOut":0,
                  "mr.respIn":0,
                  "mr.respOut":0,
                  "port":80,
                  "serverside.bitsIn":419520,
                  "serverside.bitsOut":817568,
                  "serverside.curConns":0,
                  "availabilityState":"available",
                  "enabledState":"enabled",
                  "status.statusReason":"Pool member is available"
               },
             }
         }
      }
   }
]
```

Can information related to Packets, Connections, and Request information be added. The information available by default when BIG-IP is a pull consumer is very limited.

**Comment by delgadillo22** [comment]
Created AUTOTOOL-2056 to track this in the backlog.

**Comment by dstokesf5** [comment]
This has been resolved as part of the 1.17.0 release.
